add docs for enforce mpp (#5811)
pingcap/docs: system-variables.md

- Scope: SESSION | GLOBAL
- Default value: `ON`
- This variable controls whether to use the MPP mode of TiFlash to execute queries. You can set the following values, including:

>     - `0` or `OFF`, which means never use MPP mode.

Reviewer: ```suggestion
    - `0` or `OFF`, which means that the MPP mode will not be used.
```

>     - `1` or `ON`, which means that the optimizer determines whether to use MPP mode based on the cost estimation (by default).

Reviewer: ```suggestion
    - `1` or `ON`, which means that the optimizer determines whether to use the MPP mode based on the cost estimation (by default).
```

> MPP is a distributed computing framework provided by the TiFlash engine, which allows data exchange between nodes and provides high-performance, high-throughput SQL algorithms. For details about MPP mode selection, refer to [Control whether to choose the MPP mode](/tiflash/use-tiflash.md#control-whether-to-choose-the-mpp-mode).

Reviewer: ```suggestion
MPP is a distributed computing framework provided by the TiFlash engine, which allows data exchange between nodes and provides high-performance, high-throughput SQL algorithms. For details about the selection of the MPP mode, refer to [Control whether to choose the MPP mode](/tiflash/use-tiflash.md#control-whether-to-choose-the-mpp-mode).
```

- Default value: `ON`
- This variable is used to control whether to enable the support for window functions. Note that window functions may use reserved keywords. This might cause SQL statements that could be executed normally cannot be parsed after upgrading TiDB. In this case, you can set `tidb_enable_window_function` to `OFF`.

### `tidb_enforce_mpp` <span class="version-mark">New in v5.1</span>

- Scope: SESSION
- Default value: `OFF`

> - This variable is used to control whether to ignore the optimizer cost estimation, and to force the use of TiFlash's MPP mode to execute the queries. You can set the following values:

Reviewer 2: 1. comma "," is usually not used before "and" in a compound sentence.
2. "to execute the queries" is a little redundant to me. Besides, "force to do sth" is more natural to me. How about "force to use TiFlash MPP mode"?

I'm not an English expert. These are only personal suggestions.

>     - `0` or `OFF`, which means MPP mode is not mandatory (default).

Reviewer 3: Maybe we can unify the expression of `default`? In `tidb_allow_mpp` it's `by default`, here is `default`.

- Default value: `ON`
- This variable is used to control whether to enable the support for window functions. Note that window functions may use reserved keywords. This might cause SQL statements that could be executed normally cannot be parsed after upgrading TiDB. In this case, you can set `tidb_enable_window_function` to `OFF`.

### `tidb_enforce_mpp` <span class="version-mark">New in v5.1</span>

- Scope: SESSION
- Default value: `OFF`

> - Controls whether to ignore the optimizer's cost estimation and to force queries to be executed using TiFlash's MPP mode. The value options are as follows:

Reviewer: ```suggestion
- Controls whether to ignore the optimizer's cost estimation and to forcibly use TiFlash's MPP mode for query execution. The value options are as follows:
```

> - This variable is used to control whether to ignore the optimizer cost estimation and force queries to be executed using TiFlash's MPP mode. You can set the following values:

Reviewer: ```suggestion
- Controls whether to ignore the optimizer's cost estimation and to force queries to be executed using TiFlash's MPP mode. The value options are as follows:
```

>     - `0` or `OFF`, which means MPP mode is not mandatory (by default).

Reviewer: ```suggestion
    - `0` or `OFF`, which means that the MPP mode is not forcibly used (by default).
```

>     - `1` or `ON`, which means that the cost estimation is ignored and MPP mode is forced to use. Note that this setting only takes effect when `tidb_allow_mpp=true`.

Reviewer: ```suggestion
    - `1` or `ON`, which means that the cost estimation is ignored and the MPP mode is forcibly used. Note that this setting only takes effect when `tidb_allow_mpp=true`.
```

> MPP is a distributed computing framework provided by the TiFlash engine, which allows data exchange between nodes and provides high-performance, high-throughput SQL algorithms. For details about MPP mode selection, refer to [Control whether to choose the MPP mode](/tiflash/use-tiflash.md#control-whether-to-choose-the-mpp-mode).

Reviewer: ```suggestion
MPP is a distributed computing framework provided by the TiFlash engine, which allows data exchange between nodes and provides high-performance, high-throughput SQL algorithms. For details about the selection of the MPP mode, refer to [Control whether to choose the MPP mode](/tiflash/use-tiflash.md#control-whether-to-choose-the-mpp-mode).
```
